conda-forge.yml

```diff
@@ -7,3 +7,8 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+noarch_platforms:
+  - linux_64
+  - linux_aarch64
+  - linux_ppc64le
```

Comment:
Maybe we can drop these as well since the Linux case will cover them all with `noarch`

```suggestion
```